pom.xml

```diff
@@ -109,17 +110,17 @@
             <version>2.0.2-beta</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
+        <dependency> <!-- TODO why is this here? -->
```

Comment:
Either I added this dependency for some test that was failing, or I copied it from another plug-in while updating the pom.xml (very likely). Happy to remove it if not necessary (i.e. if build passes without it, or dependency tree shows it's pointless to include it)

Reply:
Not sure offhand. Leaving it there for purposes of this PR.